Dining/models.py

```diff
@@ -330,10 +282,10 @@ def get_latest_visit(cls, dining_list, user, update=False):
         return timestamp
 
 
-class DiningDayAnnouncements(models.Model):
+class DiningDayAnnouncement(models.Model):
     date = models.DateField()
-    title = models.CharField(max_length=15)
-    text = models.CharField(max_length=240)
+    title = models.CharField(max_length=100)
```

Comment:
I don't know why I ever set it that low haha. Thanks for clearing that from my todo list :)